lib/rock/webapp/tasks/api.rb

```diff
@@ -7,6 +8,9 @@ module Tasks
             class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
+                
+                logger.formatter = GrapeLogging::Formatters::Json.new
+                use GrapeLogging::Middleware::RequestLogger, {logger: logger}
```

Comment:
That is not so easy, as grape itself is evaluating the class, not the instance, one can move the code to tasks.rb, before mount, but setting it as a variable or in initialize() won't work. 

Having two API classes in tasks.rb also doesn't work, as grape only allows to mount the API once:

```ruby
        module Tasks
            # Root for a REST API that allows to access a running Rock system
            class Root < Grape::API
                version 'v1'
                mount API
            end
            
        end
        
        module LogTasks
            # Root for a REST API that allows to access a running Rock system
            class RootLog < Grape::API

                logger.formatter = GrapeLogging::Formatters::Json.new
                use GrapeLogging::Middleware::RequestLogger, {logger: logger}

                mount API
            end        
```

Global vars are also not working.

The only way I found to make it optional is rather ugly: by using an environment variable that is set before executing the script (setting the ENV in the rock-rest-api bin also doesn't work.)